src/hatch/config/model.py

```diff
@@ -378,6 +379,29 @@ def python(self, value):
         self.raw_data['python'] = value
         self._field_python = FIELD_TO_PARSE
 
+    @property
+    def working_directory(self):
+        if self._field_working_directory is FIELD_TO_PARSE:
+            if 'working_directory' in self.raw_data:
+                working_directory = self.raw_data['working_directory']
+                if not isinstance(working_directory, str):
+                    self.raise_error('must be a string')
+
+                valid_options = ('project', 'current')
```

Comment:
shoud this be defined in the `constants` module?

Reply:
I don't think so. Other options don't use `constants` module